Sprint-1/destructuring/exercise-2/exercise.js

```diff
@@ -70,3 +70,16 @@ let hogwarts = [
     occupation: "Teacher",
   },
 ];
+
+function createListGryffindor(hogwarts) {
+  const listGryffindor = hogwarts.filter(({ house }) => house === "Gryffindor").map(({ firstName, lastName }) => `${firstName} ${lastName}`);
```

Comment:
You are creating this function and parameters. What do you think createListGryffindor() will accept as parameters? Sure, when you will invoke this method, you will pass hogwarts to it (as this variable is defined for you), but it is up to you to come up with a good name for the parameters of your function. Think what this parameter will hold as a value and name it accordingly. This is not a show-stopper for your PR but it is good to practice.
Next question, what will happen if I will do something like this:

createListGryffindor(null);

Reply:
Thank you @tyzia for reviewing my task.
Yes, you are absolutely right. It's better to use more general names for a function's parameters. I changed the parameter to 'memberList.'

If I call the function with a `null` argument, I'll get a TypeError: Cannot read properties of null.
I added a condition to check if the parameter of the function is an array.